src/orange/OrangeTrackView.hh

```diff
@@ -506,9 +507,18 @@ CELER_FUNCTION VolumeInstanceId OrangeTrackView::volume_instance_id() const
     // Otherwise we're in a background volume, and the volume instance in the
     // parent level *must* be a volume instance if this is a correctly
     // constructed geometry
-    CELER_ASSERT(lev != LevelId{0});
+    CELER_ASSERT(lev != orange_global_level);
     --lev;
-    return get_vol_inst();
+    return get_vol_inst(lev);
+}
+
+//---------------------------------------------------------------------------//
+/*!
+ * The depth in the canonical volume graph.
+ */
+CELER_FUNCTION LevelId OrangeTrackView::depth() const
+{
+    CELER_NOT_IMPLEMENTED("canonical depth");
```

Comment:
This will never be implemented, since this a G4-specific metric, correct? If it's just there for the template interface it should be marked as such.